### PR TITLE
Further Introduction updates Issue #1229

### PR DESCRIPTION
**Pull request by kwright76**
Added to the introduction to include some of the topics and specific use cases that my class mates have added.

### CODE REVIEW

File: 06-process.Rmd [hunk 1]
```diff
@@ -8,6 +8,11 @@ output:
...
 
+
+In each data visualization project there are many things to consider to minimize risk and ensure a successful project. This chapter will explain many of these concepts along with some use cases that can be utilized for specific types of businesses. One of the critical topics that is explored is risk, as minimizing risk is a key factor when deciding what data to utilize and how a specific chart type would characterize the date best. Along with risk there are certain limitations a team could face that do not pertain to data. The people and skills that are a part of the team need to be considered as this could limit what audience the visualization could be presented to. For example a tableau user would most likely not have the skills to use Altair let alone D3. These are just a few examples of items that will be found in the Health, Finance, and Retail use cases explained in the chapter.
```
Comment (syragrg) [inline 06-process.Rmd]: did you mean data at the third sentence?

Comment (syragrg) [reply 06-process.Rmd]: could you put a comma after 'For example' and Altair @ second last sentence

**Review by syragrg — CHANGES_REQUESTED** [review]
Hi Kyle just some minor grammatical and spelling issues need to be fixed @ third and second last sentence! other than that, great addition to help with the flow of the chapter!

**Comment by mschermann** [comment]
@kwright76 Please respond to reviews and merge.

**Comment by kwright76** [comment]
Hi @syragrg, I went ahead and made the corrections. Thanks for the review!

**Review by syragrg — APPROVED** [review]
thank you Kyle!

**Pull request merged**